lib/internal/webstreams/readablestream.js

```diff
@@ -931,12 +933,15 @@ class ReadableStreamBYOBReader {
 
   /**
    * @param {ArrayBufferView} view
+   * @param {{
+   *   min? : number
+   * }} [options]
    * @returns {Promise<{
    *   view : ArrayBufferView,
    *   done : boolean,
    * }>}
    */
-  read(view) {
+  read(view, options = kEmptyObject) {
```

Comment:
```suggestion
  async read(view, options = kEmptyObject) {
```
this should simplify all the reject cases to be a throw.